client/views/msg_preview_toggle.tpl

```diff
@@ -0,0 +1 @@
+<button class="toggle-button opened" data-url="{{url}}" aria-label="Toggle prefetched media"></button>
```

Comment:
Pretty sure we are able to change text depending on type, so `Toggle image` and `Toggle link preview`?